.github/workflows/release.yml

```diff
@@ -33,21 +34,19 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
 
-      - name: Publish release
```

Comment:
I see no usages of release-drafter actions anymore. It's not required as now the CI is triggered by the release itself?

Reply:
Correct - before the release would be triggered by pushing the tag. Now it's the opposite